scripts/release.sh

```diff
@@ -244,7 +297,11 @@ if [[ $? == 1 ]]; then
     # We pass in the version so that the framework script does not need to ask
     # Give the user permissions to the framework script, then run the script.
     chmod u+x ./scripts/create_framework.sh
-    ./scripts/create_framework.sh $new_version_number
+    if [ $(uname -m) == "x86_64" ]; then
+        ./scripts/create_framework.sh $new_version_number
+    else
+        arch -x86_64 /bin/bash ./scripts/create_framework.sh $new_version_number
```

Comment:
This is not working on M1 at least:
<img width="222" alt="Screen Shot 2022-07-28 at 9 35 52 AM" src="https://user-images.githubusercontent.com/1767063/181518674-c6e6d6cf-fcc2-4d72-8f12-cc347966a2f8.png">

Reply:
I think I have fixed this by using tar instead of zip.